### ISSUE TITLE
🐞[BUG]: NGXS Storage plugin breaks navigation

### ISSUE THREAD

**Issue by behoyh** [ngxs/store#1480]
### Affected Package

 The issue is caused by package @ngxs/router-plugin

### Is this a regression?

 Yes, the previous version in which this bug was not present was: <3.5.0

### Description

Repost of [this](https://stackoverflow.com/questions/58733259/ngxs-storage-plugin-breaks-navigation) SO question, since I am experiencing the same issue...

> I am wondering, why the ngxs redirects to homepage when another page is being reloaded? I see in console RouterCancel events. It happens only if ngxs/storage plugin is installed. To reproduce: go to Profile page and click the browser page reload button. Here is an example on (StackBlitz)[https://stackblitz.com/edit/ngxs-repro-4x6vcv] 
I use the latest version of ngxs/storage plugin which is 3.5.1. There are similar questions on the internet, but with the older version. As I found in the internet, it says that the error is fixed in version 3.5.1, but I am still facing it.
What is the actual problem?

## 🔬 Minimal Reproduction
[example](https://stackblitz.com/edit/ngxs-repro-4x6vcv)

## 🔥 Exception or Error

`[Router] RouterCancel` shown in state tree, redirected back to root  url `-> ''`

## Environment

<pre><code>
Libs:
- @angular/core version: 8.2.2
- @ngxs/store version: 3.5.1
- @ngxs/router-plugin: 3.5.1
- @ngxs/storage-plugin: 3.5.1

Browser:
- [x] Chrome (desktop) version latest
- [x] Chrome (Android) version latest
- [x] Chrome (iOS) version latest
- [x] Firefox version latest
- [x] Safari (desktop) version latest
- [x] Safari (iOS) version latest
- [x] IE version latest
- [x] Edge version latest

**Comment by splincode** [comment]
@arturovt

**Comment by behoyh** [comment]
I think the biggest problem with these plugins is just a misunderstanding of how it works. For example if a var is stored in local storage and then used to build a URL then some jankiness will occur when a user tries to visit a webpage other than the last one store example: (beshoyhanna.com) but there is really no way to handle this without setting an expiration on the storage or making an explicit storage API and putting warning for staring vars linked to the current URI.

tl;dr just deprecate the plugins, NGXS is great but the storage/router plugins add a confusion matrix on top of the system.

**Comment by markwhitfeld** [comment]
@behoyh I'm pretty sure that this issue has been resolved in the latest release (v3.6.0).
I upgraded your stackblitz and the issue went away: https://stackblitz.com/edit/ngxs-repro-vs68sn
Could you test an confirm that it is fixed for you?

I am closing this issue, please reopen if it is not resolved.

**Comment by behoyh** [comment]
Looks like it's fixed! Thank you!!
